go-controller/pkg/ovn/master_test.go

```diff
@@ -126,6 +127,7 @@ func cleanupGateway(fexec *ovntest.FakeExec, nodeName string, nodeSubnet string,
 	cleanupPBRandNATRules(fexec, nodeName, []*net.IPNet{ovntest.MustParseIPNet(nodeSubnet)})
 }
 
+/*
```

Comment:
I decided not to. This code is referenced from a another commented section with a FIXME

https://github.com/ovn-org/ovn-kubernetes/blob/74ac681eac72a1fa275b23ed404ae84f1f1d28fb/go-controller/pkg/ovn/master_test.go#L409

So I didn't remove it in case it is useful as reference for that FIXME, at least until we have the relevant code migrated to libovsdb.

Reply:
I think the commented code in `master_test.go` will not be addressed ever. It was commented almost one year ago. But fine, let's leave it like this for now and use a dedicated PR for cleaning that up / discussing it